### DISCUSSION TITLE
Need your help now!

### DISCUSSION THREAD

**Discussion by EasyRuby** [Monika-After-Story/MonikaModDev#8732] [general - submod / external content]
Hey guys,maybe I need your little help.
I want to change monika's room into other styles，can you provide some mods to me?
Thanks so much for your attention and help!

**Comment by janneslessmann** [comment]
You may look at this link from the wiki.
https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Custom-Backgrounds

**Comment by EasyRuby** [comment]
> 你可以从维基上查看这个链接。 https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Custom-Backgrounds

Thanks，bro，I just need it！
